### ISSUE TITLE
Can you add an option to disable the shadow? Thank you!

### ISSUE THREAD

**Issue by yip924** [ddddxxx/LyricsX#155]

**Comment by ddddxxx** [comment]
I think it would be better to let you set progress color and shadow color separately, and enable alpha channel of shadow color.

**Comment by yip924** [comment]
That's true! Thanks for considering this suggestion. Keep up the good work. Looking forward to the new update!

**Comment by ddddxxx** [comment]
[enjoy!](https://github.com/ddddxxx/LyricsX/releases/tag/v1.4.0)
